block_visualizer_plugin/pyproject.toml

```
@@ -0,0 +1,23 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "block_visualizer_plugin"
+author = [
+    {name = "Bojan Živanić", email = "zivanic.sv61.2021@uns.ac.rs"}
+]
+description = "Block visualizer plugin for visualization of the graph"
+requires-python = ">=3.9"
+classifiers = [
+    "Programming Language :: Python :: 3",
+]
+dependencies = [
+    "graph_visualizer_api",
+    "os",
+    "jinja2",
+    "d3"
+]
+[project.entry-points."gv.plugins.visualizer"]
+block = "block_visualizer_plugin.visualizer:BlockVisualiser"
```

There is a small typo of the class name, and the module names should match.

```suggestion
block = "block_visualizer_plugin.block_visualizer:BlockVisualizer"
```